ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -845,24 +845,26 @@ spec:
           workspace: repository
           subPath: src
 
-    - name: add-fbc-fragments-to-index
+    - name: build-fbc-index-images
       runAfter:
         # This is just a temporary location until we have a proper
         # FCB branch based on config
```

Comment:
Not related to the logic, but there are multiple comments accross files with this "temporary location" - if there are no further plans to move the tasks, shouldn't this be removed?

Reply:
I removed these comments.